conventions/src/main/kotlin/otel.spotless-conventions.gradle.kts

```diff
@@ -18,7 +18,7 @@ spotless {
     groovy {
       licenseHeaderFile(
         rootProject.file("buildscripts/spotless.license.java"),
-        "(package|import|class)"
+        "(package|import|(?:abstract )?class)"
```

Comment:
this was failing on `Log4J2BaggageTest.groovy` which is an abstract class with no package/imports